docs/install/install-from-packages.md

```diff
@@ -166,7 +166,10 @@ Install the latest Yarn 3 version (not the Classic 1.x one) using `npm`.
 1.  Open a terminal and type:
 
     ```shell
-    npm install yarn@3
+    npm install yarn
```

Comment:
Which version of yarn does this install? I think this installs the latest, currently 4.0.2, and I don't know if that is desired at this moment.